deliverables/README.md

```diff
@@ -32,37 +37,106 @@ What is the role of Javascript in a web page?
 #### meta
 Why do we separate HTML and CSS?  
 
+1. You can apply the same formatting o several HTML elements without rewriting code over and over.  
+2. You can apply appearence and formatting to several HTML pages from a single CSS file.  
+
 #### HTML
 What are classes and IDs (and how are they different)?  
+HTML uses Classes and IDs to refer to CSS.  
+
 What are elements?  
+Elements are individual components of HTML. They are written with an opening tag and a closing tag. There is content in between the tags.  
+
 What are tags?  
+Tags are used to code elements. There are tags for comments, hyperlinks, paragraphs and many more. They are inside <>s.  
+
 What are attributes?  
+HTML elements have attributes, they provide additional information about an element. We use attributes for titles, colors, fonts and a lot more!  
+
 What are forms?  
+Forms are used to collect user input.  
+
 What is a div?  
+A div is a tag that defines a division or a setion in an HTML document.  
 
 What are selectors?  
+Selectors are patterns used to select the elements you want to style.  
+
 What are properties?  
+Property is an aspect of a selector. For example changing the font, color, or font size.  
+
 What are values?  
+A value us a possible setting for a property.  
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+Styles are automatically inherited by child elements from their parent elements, by an element contained inside another one.   
+
 What are two CSS attributes you can change to push an element around on the page?  
+Float is an element that can can be pushed to the left or right, allowing other elements to wrap around it.  
+
 What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+Using the element selector, id selector or class selector.
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.  
+
 What is SCM?  
+SCM is  
+
 What is a VCS?  
+VCS is a version control system  
+
 Why is Git useful for a developer?  
+It is free and open source, and can keep track of all of your projects.  
```

Comment:
Free and open source is great! It's also very useful in allowing us to keep track of changed in our code - really important when we make break something - and a great way to try out new features without worrying about creating issues.